dll/win32/urlmon/urlmon_main.h

```diff
@@ -24,6 +24,8 @@
 
 #define COBJMACROS
 
+#define PROXY_CLSID_IS {0x79EAC9F1,0xBAF9,0x11CE,{0x8C,0x82,0x00,0xAA,0x00,0x4B,0xA9,0x0B}}
```

Comment:
This file seems to be in sync with Wine, so `#ifdef __REACTOS__` might be appropriate.
Or is there some way we can keep this in the CMake file?

Reply:
It's in sync with wine yeah, will add an #ifdef
Making a similar change to upstream wine might be a better solution, it seems to be the only class id stored as a compiler argument